### PR TITLE
Added instructions on how to make fastlane fast again

### PR DESCRIPTION
**Pull request by KrauseFx**
This will add a detection of how slow fastlane currently is, and instructions on how to make it fast again

I'll add that it will detect if there is already a `Gemfile`, if there is, we should tell the user to use `bundle exec`

### CODE REVIEW

**Comment by neonichu** [comment]
:+1:

**Comment by orta** [comment]
Optional, you could include a link to explain what bundler is - https://guides.cocoapods.org/using/a-gemfile.html

File: fastlane/lib/fastlane/cli_tools_distributor.rb [hunk 1]
```diff
@@ -67,6 +67,7 @@ def print_slow_fastlane_warning
           UI.important "to launch fastlane faster, please use"
           UI.message ""
           UI.command "bundle exec fastlane #{ARGV.join(' ')}"
+          UI.important "For more information, check out https://guides.cocoapods.org/using/a-gemfile.html"
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/cli_tools_distributor.rb]: cc @orta

**Comment by KrauseFx** [comment]
Thanks @orta, added this to the PR

**Pull request merged**